check_py_coverage.sh

```diff
@@ -65,6 +65,33 @@ do
     coverage run $EXTRA $DRIVER $I
 done
 coverage run $EXTRA ../../framework/TestDistributions.py
-coverage run $EXTRA ../../framework/Driver.py test_relap5_code_interface.xml interfacecheck
+
+# code interface tests START
+cd CodeInterfaceTests
+coverage run $EXTRA $FRAMEWORK_DIR/Driver.py test_relap5_code_interface.xml interfaceCheck
+coverage run $EXTRA $FRAMEWORK_DIR/Driver.py test_perturb_mammoth_bison_relap7.xml interfaceCheck
+coverage run $EXTRA $FRAMEWORK_DIR/Driver.py test_mammoth_r7_bison_no_exe_hdf5_restart.xml interfaceCheck
+coverage run $EXTRA $FRAMEWORK_DIR/Driver.py test_perturb_all_rattlesnake_bison.xml interfaceCheck
+coverage run $EXTRA $FRAMEWORK_DIR/Driver.py test_perturb_mammoth_rattlesnake_bison.xml interfaceCheck
+coverage run $EXTRA $FRAMEWORK_DIR/Driver.py test_relap5_code_interface_alias.xml interfaceCheck
+coverage run $EXTRA $FRAMEWORK_DIR/Driver.py test_relap5_code_interface_multideck.xml interfaceCheck
+coverage run $EXTRA $FRAMEWORK_DIR/Driver.py test_relap5_code_interface_multideck_choosing_deck_output.xml interfaceCheck
+coverage run $EXTRA $FRAMEWORK_DIR/Driver.py test_OpenModelica_code_interface.xml interfaceCheck
+coverage run $EXTRA $FRAMEWORK_DIR/Driver.py test_Dymola_code_interface.xml interfaceCheck
+coverage run $EXTRA $FRAMEWORK_DIR/Driver.py test_Dymola_code_interface_timedep.xml interfaceCheck
+coverage run $EXTRA $FRAMEWORK_DIR/Driver.py test_maap5_code_interface_forward.xml interfaceCheck
+coverage run $EXTRA $FRAMEWORK_DIR/Driver.py test_maap5_code_interface_det.xml interfaceCheck
+coverage run $EXTRA $FRAMEWORK_DIR/Driver.py test_maap5_code_interface_hybrid_det.xml interfaceCheck
+coverage run $EXTRA $FRAMEWORK_DIR/Driver.py test_maap5_code_interface_adaptive_det.xml interfaceCheck
+coverage run $EXTRA $FRAMEWORK_DIR/Driver.py test_maap5_code_interface_adaptive_hybrid_det.xml interfaceCheck
+coverage run $EXTRA $FRAMEWORK_DIR/Driver.py test_maap5_code_interface_det_multibranch.xml interfaceCheck
+coverage run $EXTRA $FRAMEWORK_DIR/Driver.py test_relap5_code_inss.xml interfaceCheck
+# code interface tests END
+cd ../PostProcessors/TopologicalPostProcessor
+coverage run $EXTRA $FRAMEWORK_DIR/Driver.py test_topology_ui.xml interactiveCheck
+cd ../DataMiningPostProcessor/Clustering/
+coverage run $EXTRA $FRAMEWORK_DIR/Driver.py hierarchical_ui.xml interactiveCheck
+cd ../../../../../
```

Comment:
@joshua-cogliati-inl, I ran this several times on my personal machine, but I was typically running a piece of this file and not the whole thing. It may be worthwhile to ensure that this last line is stepping out to the appropriate directory. In fact, this line should probably reset the working dir:

```bash
cd $FRAMEWORK_DIR/..
```

Rather than try to keep track of where it is at.